scipy/integrate/tests/test_ivp.py

```diff
@@ -296,6 +296,29 @@ def test_integration_const_jac():
         assert_allclose(res.sol(res.t), res.y, rtol=1e-14, atol=1e-14)
 
 
+def test_integration_stiff():
+    rtol = 1e-6
+    atol = 1e-6
+    y0 = [1e4, 0, 0]
+    tspan = [0, 1e8]
+
+    def fun_robertson(t, state):
+        x, y, z = state
+        return [
+            -0.04 * x + 1e4 * y * z,
+            0.04 * x - 1e4 * y * z - 3e7 * y * y,
+            3e7 * y * y,
+        ]
+
+    for method in ['Radau', 'BDF', 'LSODA']:
```

Comment:
pytest parametrization might be nice here, but some people tell me "no" when I ask for that anyway; certainly not crucial

Reply:
>  but some people tell me "no" when I ask for that

Tyler, if that was me, I'd say "yes" now.  Consider me enlightened (or just trained :), and thanks for being persistent about it.

Reply:
I switched this test to pytest parameterization.